tests/src/test/scala/runtime/actionContainers/Python39ctionLoopContainerTests.scala

```diff
@@ -24,12 +24,12 @@ import spray.json._
 import DefaultJsonProtocol._
 
 @RunWith(classOf[JUnitRunner])
-class Python3AiActionLoopContainerTests
+class Python39ActionLoopContainerTests
```

Comment:
shouldn't this be a copy? you are removing the ai tests?

Reply:
cleaned all the tests